package.json

```diff
@@ -19,7 +19,8 @@
     "lint": "standard",
     "lint:fix": "standard --fix",
     "generate": "node scripts/run.js",
-    "elasticsearch": "docker run --rm -e \"node.attr.testattr=test\" -e \"path.repo=/tmp\" -e \"repositories.url.allowed_urls=http://snapshot.*\" -p 9200:9200 docker.elastic.co/elasticsearch/elasticsearch:6.4.0"
+    "elasticsearch": "docker run --rm -e \"node.attr.testattr=test\" -e \"path.repo=/tmp\" -e \"repositories.url.allowed_urls=http://snapshot.*\" -p 9200:9200 docker.elastic.co/elasticsearch/elasticsearch:6.4.0",
+    "ci": "./scripts/wait-cluster.sh && npm test && npm run test:integration"
```

Comment:
So trying to run this locally failed cause the `wait-cluster.sh` script is refrencing `localhost:9200` vs `http://elasticsearch:9200`. 

here's what I did and things work... not sure how you use this script otherwise... so maybe we need to check if `TEST_ES_SERVER` env var exists first and maybe default it? 

```bash
#!/bin/bash

attempt_counter=0
max_attempts=5
url="${TEST_ES_SERVER}/_cluster/health?wait_for_status=green&timeout=50s"

echo "Waiting for Elasticsearch..."
echo $url
while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' --max-time 55 "$url")" != "200" ]]; do
  if [ ${attempt_counter} -eq ${max_attempts} ];then
    echo "\nCouldn't connect to Elasticsearch"
    exit 1
  fi

  printf '.'
  attempt_counter=$(($attempt_counter+1))
  sleep 5
done

echo "\nReady"
```